include/fmt/format.h

```diff
@@ -278,9 +278,879 @@ FMT_END_NAMESPACE
 #  define FMT_DEPRECATED_NUMERIC_ALIGN 0
 #endif
 
+#define FMT_POWERS_OF_10(factor)                                             \
+  factor * 10, (factor)*100, (factor)*1000, (factor)*10000, (factor)*100000, \
+      (factor)*1000000, (factor)*10000000, (factor)*100000000,               \
+      (factor)*1000000000
```

Comment:
Why turn this and other data into macros?

Reply:
Maybe I overdid it, but I think that some more entities from there will be _exposed_  to use them at compile-time later. Like data for floating-point formatting, hex and maybe other.

Reply:
FP data won't need to move, so please revert those changes.

Reply:
Done, but some of them will return to `format.h` later.